[tests] Ignore the MonoTouchFixtures.ObjCRuntime.RegistrarTest.RegistrarRemoval test, it's failing on .NET. Fixes #xamarin/maccore@2380.

rolfbjarne opened the pull request:
Fixes https://github.com/xamarin/maccore/issues/2380.

vs-mobiletools-engineering-service2 commented:
### :x: Tests failed on Build :x:

Tests failed on Build. 
* [Azure DevOps](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=4403281&view=ms.vss-test-web.test-result-details)
* [Html Report (VSDrops)](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210125.5/4403281/sim/;/tests/vsdrops_index.html)


# Test results

<details>
<summary>10 tests failed, 165 tests passed.</summary>

## Failed tests

 * introspection/Mac Modern/Debug: BuildFailure
 * introspection/watchOS 32-bits - simulator/Debug (watchOS 3.2): Failed
 * framework-test/Mac Modern/Debug: BuildFailure
 * xcframework-test/Mac Modern/Debug: BuildFailure
 * [xUnit] Mono Mac OS X BCL tests group 4/Mac Modern/Debug: Failed (No test log file was produced)
 * link sdk/iOS Unified 64-bits - simulator/Debug [dotnet]: Failed
 * link sdk/iOS Unified 64-bits - simulator/Release [dotnet]: Failed
 * mmptest/macOS/Debug: Failed (Execution failed with exit code 1)
 * MSBuild tests/Integration: Failed (Execution failed with exit code 31)
 * MTouch tests/NUnit: Failed (Execution failed with exit code 7)
</details>


[Pipeline](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=4403281&view=ms.vss-test-web.test-result-details) on Agent XAMBOT-1097

vs-mobiletools-engineering-service2 commented:
### :x: Tests failed on Build :x:

Tests failed on Build. 
* [Azure DevOps](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=4412859&view=ms.vss-test-web.test-result-details)
* [Html Report (VSDrops)](https://vsdrop.corp.microsoft.com/file/v1/xamarin-macios/device-tests/20210127.10/4412859/sim/;/tests/vsdrops_index.html)


# Test results

<details>
<summary>3 tests failed, 172 tests passed.</summary>

## Failed tests

 * link sdk/iOS Unified 64-bits - simulator/Debug [dotnet]: Failed
 * link sdk/iOS Unified 64-bits - simulator/Release [dotnet]: Failed
 * introspection/watchOS 32-bits - simulator/Debug (watchOS 3.2): Failed
</details>


[Pipeline](https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=4412859&view=ms.vss-test-web.test-result-details) on Agent XAMBOT-1094'

rolfbjarne commented:
Test failures are unrelated.

* link sdk/iOS Unified 64-bits - simulator/Debug [dotnet]: Fixed in #10511.
* link sdk/iOS Unified 64-bits - simulator/Release [dotnet]: Fixed in #10511.
* introspection/watchOS 32-bits - simulator/Debug (watchOS 3.2): https://github.com/xamarin/maccore/issues/2378
